asab/config.py

```diff
@@ -188,35 +181,27 @@ def _load(self):
 
 		del self._load_dir_stack
 
-	def _from_include_build_url(self, purl):
-
-		parse_include = urlparse(purl)
-		url_path = parse_include.path
-		url_scheme = parse_include.scheme
-		url_netloc = self["asab.zookeeper"]["url"]
-
-		if purl.rfind("///") == 10:
-			url_comp = (url_scheme, url_netloc, url_path, '', '', '')
-
-		elif purl.rfind(":.") == 9:
-			new_url_path = self["asab.zookeeper"]["path"] + url_path[1:]
-			url_comp = (url_scheme, url_netloc, new_url_path, '', '', '')
-
-		#unparse  components into proper URL
-		url_built = urlunparse(url_comp)
-		return  url_built
 
 	def _include_from_zookeeper(self, zkurl):
 		import aiozk
 
 		loop = asyncio.get_event_loop()
+
 		#parse include value into hostname and path
 		url_pieces = urlparse(zkurl)
 		url_path = url_pieces.path
 		url_netloc = url_pieces.netloc
 
+		if not url_netloc:
+			url_netloc = self["asab:zookeeper"]["servers"]
+
+		if url_path.startswith("./"):
+			url_path = self["asab:zookeeper"]["path"] + url_path[1:]
+
 		async def download_from_zookeeper():
 			try:
+				root_logger = logging.getLogger()
+				root_logger.disabled = True
```

Comment:
Adjust please the closing so that we don't need to manipulate root logger log level.